schemas/launchconfiguration.schema.tpl.json

```diff
@@ -0,0 +1,35 @@
+{
+    "_type": "https://openminds.ebrains.eu/computation/LaunchConfiguration",
```

Comment:
note: we keep lowerCaseCamel for the filenames as well.

Reply:
unclear what this refers to, Github is showing me "_type", which does not reference a filename